client/settings/digital-wallets/index.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { __ } from '@wordpress/i18n';
+import {
+	Button,
+	Card,
+	CardBody,
+	CardDivider,
+	CheckboxControl,
+} from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+
+const DigitalWallets = () => {
+	const [ isEnabled, setIsEnabled ] = useState( false );
+	const [ sectionsStatus, setSectionsStatus ] = useState( {
+		checkout: true,
+		productPage: true,
+		cart: true,
+	} );
+
+	const makeHandleSectionStatusChange = ( section ) => ( status ) => {
```

Comment:
Yeah, I find myself using the curry pattern quite often -  that's why I prefixed with `make` (since it's not a handler) :) It can be quite useful!
I don't know if it would be easier 👀  I guess I'm kinda used to it now 😄  I noticed the same pattern was also added to `client/settings/payment-methods-selector/index.js` so I thought it would be ok as is 🤷

Reply:
Definitely looks great -- it's just that I'm not used to it so I started wondering if I should _get_ used to it or if maybe it'd be clearer to use something else for the sake of clarity. 😄 The solution itself is very clean, GG!